test/CMakeLists.txt

```diff
@@ -25,6 +25,15 @@ endif()
 file(TO_CMAKE_PATH "${PROJECT_SOURCE_DIR}/valgrind.supp" MEMORYCHECK_SUPPRESSIONS_FILE_TMP)
 set(MEMORYCHECK_SUPPRESSIONS_FILE ${MEMORYCHECK_SUPPRESSIONS_FILE_TMP} CACHE FILEPATH "Valgrind suppresions file")
 
+# Check if /etc/hosts file has been modified to add dummy host test subjects
+if(WIN32)
+    execute_process(COMMAND powershell -C Resolve-DNSName www.acme.com.test
```

Comment:
Not sure this was intented for this PR. Please check.

Reply:
see above